ChiModules/LinearBoltzmannSolver/lbs_01d_init_spat_discr.cc

```diff
@@ -20,10 +19,112 @@ void lbs::SteadySolver::InitializeSpatialDiscretization()
     chi_math::SpatialDiscretization_PWLD::New(grid, COMPUTE_CELL_MAPPINGS |
                                                     COMPUTE_UNIT_INTEGRALS);
 
+  ComputeUnitIntegrals();
+}
+
+void lbs::SteadySolver::ComputeUnitIntegrals()
```

Comment:
The fact that we do this here essentially allows us to phase-out the computing of unit integrals by the spatial discretizations.